### ISSUE TITLE
New UI feedback, and problems.

### ISSUE THREAD

**Issue by CimkNoYAP** [modrinth/code#3061]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [x] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

### What parts of Modrinth is your feature request related too?

App

### Is your suggested feature related to a problem? Please describe.

I really wanted to give feedback to the new ui on the app. So everything is really nicely organized and looks super clean. But there are some problems with the oversimplicity.

### Describe the solution you'd like

So lets get to the solutions! First of all: the mod delete button. it is hidden all the way in the three dots on a mod. It would be better if we could do it as a simple button next to the disable button. Second of all: the install content button, I feel like it would be better if it would have that nice green color that you use to show what page your on, or the play button, etc... Third of all: the export modpack button. It should be somewhere like below the name of the modpack or next to it. As i always love to export my modpack to my friend on discord. same with the open modpack folder, it should just be somewhere. Also, while writing this, I looked at the modrinth app, and saw that one of the modpack names where out of the gray box, instead of dotting at the end, it just escaped the box. Maybe bug-fix that a little, but that's really it! I enjoy the new UI a lot and it looks really clean.!     

[Image](https://github.com/user-attachments/assets/8b6b0d55-90bc-4ada-b91b-f750c2a66541)

### Additional context

![Image](https://github.com/user-attachments/assets/ae797595-9de7-4a4c-ab31-ecbf3d4cf924)

my very nice artwork that shows you what it could potentially look like.

**Comment by CimkNoYAP** [comment]
oh wait, this is in the issue, not pull request, oops. Well I would still like the fixes I issued up there.

**Comment by BoraMD** [comment]
Also it would be nice to see is it client only or client and server etc. So i can just update my client mods without updating mods on the server too. At old ui i can just click the mod and see if it is client or not isnt the easiest way but at least i can. In the new version i cant see even i click on mod.

**Comment by achookh** [comment]
1)i think the titles above the table should become part of it
2)please bring back filtering by author
3)content type would be a good addition
4)we need to be able to change how much items we view (like in discovery)
5)page numbers always at the right side (it gets moved by filters anyway)
6)refresh button on the left before the filters (it will look weird inside the table)

![Image](https://github.com/user-attachments/assets/e62c481d-da6c-4113-8d70-66ca89b0a3fd)

**Comment by Felix14-v2** [comment]
> And also the fact that the close button makes you aim is diabolical

You can enable native decorations in the launcher settings to fix that

**Comment by CimkNoYAP** [comment]
yeah... lets hope they fix these simple issues
they already fixed quite a lot of them

**Comment by RubyCascade** [comment]
I agree the Install Content button needs to be green, the enable/disable button needs to be reverted to a toggle switch, and that the trash button needs to be returned to not being in a menu.
I also agree with adding client side, server side, either, and both _(or just the first two, idk)_ filter options to the filter for the modpack.

My gripe: add back the thing that says if a mod is client side, server side, client and server side, or client or server side to the mod's page both in the app and on the website. _(For the past couple days I had to open the website to see if a mod is client or server side, but now I can't even do that. Why was it removed in the first place?)_

Also, please add "Copy mod link" beneath the "Open in browser" button on the mod's page in the app.

**Comment by Minenash** [comment]
> oh wait, this is in the issue, not pull request, oops. 

Issues is still correct here, pull requests are more "here are code changes, please apply them"

> The top bar has plenty of space for buttons anyway

The app is designed in mind with the smallest window. You can see here that there isn't that much extra room. That being said, having it show if there's space sounds good, because I agree with you. I love the dedicated button for the folder.

![Image](https://github.com/user-attachments/assets/096490ba-0a5a-4cf4-ba44-2b50aff691ca)

**Comment by Prospector** [comment]
Please keep issues to one topic per issue
